From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Encode42 <me@encode42.dev>
Date: Thu, 16 Sep 2021 20:32:04 -0400
Subject: [PATCH] Tentacles config files


diff --git a/src/main/java/org/bukkit/Server.java b/src/main/java/org/bukkit/Server.java
index ccb8909d3da0bb6ccc6d35e8497c3b4b993b9044..72f0f6685a2681a7f8c11bdd9a7be268c6e0f132 100644
--- a/src/main/java/org/bukkit/Server.java
+++ b/src/main/java/org/bukkit/Server.java
@@ -1912,6 +1912,13 @@ public interface Server extends PluginMessageRecipient, net.kyori.adventure.audi
         }
         // Purpur end
 
+        // Tentacles start
+        @NotNull
+        public org.bukkit.configuration.file.YamlConfiguration getTentaclesConfig() {
+            throw new UnsupportedOperationException("Not supported yet.");
+        }
+        // Tentacles end
+
         /**
          * Sends the component to the player
          *
