From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Encode42 <me@encode42.dev>
Date: Thu, 16 Sep 2021 20:32:04 -0400
Subject: [PATCH] Tentacles config files


diff --git a/src/main/java/org/bukkit/Server.java b/src/main/java/org/bukkit/Server.java
index e15b869241113633ed0ded1d49c805ef7b40460c..9c724e5cf6d5e2081c43211d67e6f974334c1908 100644
--- a/src/main/java/org/bukkit/Server.java
+++ b/src/main/java/org/bukkit/Server.java
@@ -1914,6 +1914,13 @@ public interface Server extends PluginMessageRecipient, net.kyori.adventure.audi
         }
         // Purpur end
 
+        // Tentacles start
+        @NotNull
+        public org.bukkit.configuration.file.YamlConfiguration getTentaclesConfig() {
+            throw new UnsupportedOperationException("Not supported yet.");
+        }
+        // Tentacles end
+
         /**
          * Sends the component to the player
          *
